### ISSUE TITLE
execute same sql but use jdbc sometimes appears error

### ISSUE THREAD

**Issue by xfdFan** [ClickHouse/clickhouse-java#549]
I am using ClickHouse-JDBC 0.2.4 version to connect to 20.5.4.40 service to execute SQL "select  * from table where 1=2", but occasionally the error message is "code: 497, not enough privileges", I have given the user permission, and there is no problem in clickhouse-client and dbveaver, but occasionally this problem occurs when using JDBC, what is the reason? Is there a solution?

**Comment by zhicwu** [comment]
DBeaver uses the same JDBC driver. Are you saying the query didn't work in your program?

**Comment by Ow-Fan** [comment]
> DBeaver uses the same JDBC driver. Are you saying the query didn't work in your program?

yes, but it occasionally happens, not always. somestimes many executions work fine, but sometimes it happens no reason,  but I succeeded on one more try.

**Comment by zhicwu** [comment]
Is it a cluster you're connecting to? What did you observe in clickhouse server log?

**Comment by Ow-Fan** [comment]
> Is it a cluster you're connecting to? What did you observe in clickhouse server log?

yes， I connect to a cluster, clickhouse server log is the same message： "not not enough privileges". it is related to the cluster？the error is random，I test 1000 times this afternoon and it didn't happen,
